Source/cmake.cxx

```diff
@@ -589,6 +589,10 @@ void cmake::SetArgs(const std::vector<std::string>& args,
       std::cout << "Running with expanded trace output on.\n";
       this->SetTrace(true);
       this->SetTraceExpand(true);
+    } else if (arg.find("--trace-file=", 0) == 0) {
+      std::string file = arg.substr(strlen("--trace-file="));
+      cmSystemTools::ConvertToUnixSlashes(file);
+      this->AddTraceFile(file);
```

Comment:
Should this call `this->SetTrace(true);` as well? I can't think of a reason it shouldn't.